pom.xml

```diff
@@ -27,6 +27,20 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api -->
+		<dependency>
```

Comment:
It's better to avoid hardcoding the version of 'junit-jupiter-api'. Consider managing it through a properties file or a parent POM.